### PR TITLE
Add RunVirtctlCommand function

### PR DESCRIPTION
**Pull request by gouyang**
Signed-off-by: Guohua Ouyang <gouyang@redhat.com>

**What this PR does / why we need it**:
Add RunVirtctlCommand function, so it can be used to add tests like start/stop VM via this function.

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: tests/utils.go [hunk 1]
```diff
@@ -1417,6 +1416,48 @@ func RunOcCommand(args ...string) (string, error) {
...
+	if KubeVirtVirtctlPath == "" {
+		var err error
+		KubeVirtVirtctlPath, err = exec.LookPath("virtctl")
+		if err != nil {
+			Skip("Skip test that requires virtctl binary")
+		}
+	}
+}
+
+func RunVirtctlCommand(args ...string) (string, error) {
```
Comment (rmohr) [inline tests/utils.go]: Can't you use `NewRepeatableVirtctlCommand` and `NewVirtctlCommand` which are also in this file?

**Comment by rmohr** [comment]
> Add RunVirtctlCommand function, so it can be used to add tests like start/stop VM via this function.

As far as I know we already have tests for that. If something is missing, it should be possible to just add more. For instance have a look at https://github.com/kubevirt/kubevirt/blob/master/tests/vm_test.go#L435.

File: tests/utils.go (hunk 1 above)
Comment (gouyang) [reply tests/utils.go]: `NewRepeatableVirtctlCommand` and `NewVirtctlCommand` should be good enough, I didn't notice it.

**Comment by gouyang** [comment]
Some more thoughts is both `RunVirtctlCommand ` and `NewRepeatableVirtctlCommand` can start/stop vm, but the former is calling the real virtctl binary, which looks like a more e2e test.

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically.
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

**Comment by cynepco3hahue** [comment]
` M tests/utils.go
The command "if [[ -n "$(git status --porcelain)" ]] ; then echo "It seems like you need to run `make generate`. Please run it and commit the changes"; git status --porcelain; false; fi" exited with 1.`
Please run `make` and commit again.

**Comment by cynepco3hahue** [comment]
@gouyang https://github.com/kubevirt/kubevirt/pull/1408 does some generalization on command run, will it satisfy you?

**Comment by gouyang** [comment]
> #1408 does some generalization on command run, will it satisfy you?

I'm okay with that change, so I can close this one.

**Pull request closed without merging**